### ISSUE TITLE
Allow for resizing OR opening gamebots (not web-apps!) in browser

### ISSUE THREAD

**Issue by dotvhs** [telegramdesktop/tdesktop#28254]
### Is your feature request related to a problem?

Latest update changed the behavior for gamebots (like gamee for example). Games now open inside a very small (384x590) fixed-size window making it difficult to play or read text presented on them. It is a regression compared previous versions and most mobile phones have display sizes than this.

![image](https://github.com/user-attachments/assets/cbb2bb53-5883-42d6-915e-c6ea4f09103b)

### Describe the solution you'd like

Having in mind that this is a **Desktop** client, this screams for ability to resize the window in both axis. There could be size limitations for it but please allow for resizing the window.

### Describe alternatives you've considered

An alternative would be having "Open in browser" button in three dots, however this is less than ideal solution because it requires doing that every time, while resizable window could remember it's set size.

### Additional context

HTML5 Game bot documentation clearly states that games should be responsive:

> Please make sure that your HTML5 page is responsive and works well on all Telegram apps and supported platforms. 
Source: https://core.telegram.org/bots/games#creating-a-great-html5-experience

There's no clear reason for limiting game bots to fixed size, super small windows.

**Comment by tarfand121** [comment]
also some keyboard features does not work in the gamebot windows. for example see this "@gamee Run" .

**Comment by dotvhs** [comment]
> imho a lot of windows needs to be allowed resized on every direction.

Agreed but this here is a clear regression from previous behavior. Other windows just were like this from the beginning. But using game bots became now just very annoying on TDesktop.

Just because "most users use mobile anyway", we - desktop users - should not suffer from a worse experience. There was nothing wrong with previous behavior and it was superior in almost every way.

**Comment by MaxVerevkin** [comment]
+1 for allowing all windows (web-apps included) to be resized. In fact web-apps already can be resized but only on linux and only on X11, which is odd. But there resizing works and I don't see a reason why it is locked on other platforms.

**Comment by dotvhs** [comment]
Just to update on the topic: Telegram Web K allows for resizing Web Apps therefore there are even less reasons for Telegram Desktop to have this in the worst state than any other client by sheer size of its window alone.
